archway/chain.json

```
@@ -265,6 +273,10 @@
         "address": "grpc.mainnet.archway.io:443",
         "provider": "Archway Foundation"
       },
+      {
+        "address": "http://archway.grpc.nodersteam.com:9090",
```

Adress cannot have `http://` in it
```suggestion
        "address": "archway.grpc.nodersteam.com:9090",
```